cmd/bpf2go/main.go

```diff
@@ -114,11 +114,11 @@ type bpf2go struct {
 	makeBase string
 }
 
-func newB2G(stdout io.Writer, pkg, outputDir string, args []string) (*bpf2go, error) {
+func newB2G(stdout io.Writer, pkg, defaultOutDir string, args []string) (*bpf2go, error) {
```

Comment:
How about just dropping `outputDir`, and then doing the Getwd lookup in here if `*outDir == ""`?

Same for `pkg`, we can reuse `getEnv` as for `BPF2GO_CC`, etc.

Reply:
That sounds reasonable, can you help check it again?
I also add some unit tests.